packages/scss/src/components/_label.scss

```diff
@@ -1,9 +1,9 @@
 .label {
 	background: #CCCCCC;
 	border-radius: 3px;
-	color: $grey;
+	color: _get($luTheme, "palette.colors.greyStandard");
 	display: inline-block;
-	font-size: $font-small;
+	font-size: _get($luTheme, "font.size.small");
 	font-weight: 400;
 	line-height: 1;
 	margin: 0 .3rem;
```

Comment:
If every measure is rem based, that means I won't be able to make a "fat" input in my project.